### PR TITLE
2022 Q4 Hyperledger Ursa report

### PR DESCRIPTION
**Pull request by brentzundel**

### CODE REVIEW

**Comment by tkuhrt** [comment]
The Q1 report is due on March 16th. Should we make this the Q1 report? Or will you plan on releasing the Q1 report in a couple of weeks?

File: project-reports/2022/2022-Q4-Hyperledger-Ursa.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+There was very little activity this past quarter as this
+<a href="https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Fursa/dashboard;subTab=technical?time=%7B%22from%22:%222022-10-01T06:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222022-12-31T07:00:00.000Z%22%7D" rel="nofollow">Ursa Activity Dashboard</a> shows. 
+
+# Current Plans
+
+Near the end of the quarter we began to see additional engagement from some
+possible new maintainers. We also reinvigorated the discussion about
+refactoring Ursa and began to make concrete plans to that end.
+
+# Maintainer Diversity
```
Comment (tkuhrt) [inline project-reports/2022/2022-Q4-Hyperledger-Ursa.md:55]: Are these maintainers active? Or do they need to be moved to Emeritus status?

Comment (brentzundel) [reply project-reports/2022/2022-Q4-Hyperledger-Ursa.md:55]: we have recently updated the set of maintainers. Those listed there are valid for Q4, we have more now.

**Review by TimoGlastra — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Comment by tkuhrt** [comment]
> The Q1 report is due on March 16th. Should we make this the Q1 report? Or will you plan on releasing the Q1 report in a couple of weeks?

Hi, @brentzundel : We talked about this in the TOC call today. The TOC thought since there are some changes that have happened between Q4 and Q1 that it still makes sense to get the Q1 report submitted, which will be due in a couple of weeks. For the next report, you can use [this template](https://github.com/hyperledger/toc/blob/gh-pages/project-reports/0000-template.md)

**Review by tkuhrt — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Pull request merged**